src/main/clm_varctl.F90

```diff
@@ -188,6 +188,13 @@ module clm_varctl
 
   logical, public :: use_c13 = .false.                  ! true => use C-13 model
   logical, public :: use_c14 = .false.                  ! true => use C-14 model
+  !----------------------------------------------------------
+  ! CN matrix
+  !----------------------------------------------------------  
+  logical, public :: use_matrixcn = .true. !.false.              ! true => use cn matrix
+  logical, public :: use_soil_matrixcn = .true.! true => use cn matrix  
+  logical, public :: isspinup = .false.  !.false.              ! true => use acc spinup
```

Comment:
@chrislxj isspinup doesn't seem to be connected to anything or used anywhere. So it seems like there's no way to turn on the acc spinup right?

Reply:
@ekluzek Isspinup is used in soilbiogeochem/CNSoilMatrix.F90 and biogeochem/CNVegMatrix.F90. But we never test it before, so we always turn it off. Actually, it can be turned on .

I noticed there is new branch named “cn-matric_v3”, which you have added matrix items to the namelist. The branch name is different with original “cn-matrix_v3”. Should my cleanup be based on cn-matrix_v3 or cn-matric_v3?

Reply:
Hi @chrislxj thanks for that. The extra branch was a mistake. I'll remove it.